tests/mocks.py

```diff
@@ -409,6 +416,67 @@ def project(self, name, *, no_slug=False):
 
         return project
 
+    def transaction(
+        self, transaction_type: str, object: dict, operations=None, comments=None
+    ):
+        """Return a Phabricator Transaction object.
+
+        Args:
+            transaction_type: String describing the transaction type.
+                e.g. "subscribers", "projects", "accept".
+            object: A dict structured as a Phabricator API object being modified by the
+                transaction. e.g. a revision dict or project dict.
+            operations: Optional list of operations that the transaction is performing.
+                Note that this argument's value and structure can change depending on
+                the value of the "type" argument.
+                e.g. When adding a subscriber to a revision the transaction type would
+                be "subscriber" and the operations argument would be
+                [{ "operation": "add", "phid": "PHID-USER-abc123"}]
+            comments: Optional list of comments modified by this transaction. Only
+                applies when the transaction type is "comment" or "inline". Structure
+                varies greatly depending on the comment type.
+
+        """
+        # Pull out what type of object we are operating on: DREV? PROJ?
+        object_type = object["type"]
+        object_phid = object["phid"]
+        comments = comments or []
+        fields = {}
+
+        if operations:
+            fields["operations"] = operations
+
+        phid = self._new_phid(f"XACT-{object_type}-")
+        transaction = {
+            "id": self._new_id(self._transactions),
+            "phid": phid,
+            "type": transaction_type,
+            # Not implemented.
+            "authorPHID": None,
+            "objectPHID": object_phid,
+            "dateCreated": 1559779750,
+            "dateModified": 1559779750,
+            # No idea what this field is for.
+            "groupID": "zmgrbvzbcclaubtg4yt2ihwwotxewc4h",
+            "comments": comments,
+            "fields": fields,
+        }
+        self._transactions.append(transaction)
+        self._phids.append(
+            {
+                "phid": phid,
+                # Unlike other objects transactions don't have a URI. This field
+                # is just the hostname with no path component.
+                "uri": "https://phabricator.services.mozilla.com",
```

Comment:
http://phabricator.test/ is used everywhere else in this file.
```suggestion
                "uri": "http://phabricator.test/",
```